.github/workflows/release.yml

```diff
@@ -23,34 +23,22 @@ jobs:
         with:
           publish-templates: "true"
           base-path-to-templates: "./src"
-          generate-docs: "true"
 
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: Create PR for Documentation
-        id: push_image_info
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: |
-          set -e
-          echo "Start."
-
-          # Configure git and Push updates
-          git config --global user.email github-actions@github.com
-          git config --global user.name github-actions
-          git config pull.rebase false
-
-          branch=automated-documentation-update-"$GITHUB_RUN_ID"
-          git checkout -b "$branch"
-          message='Automated documentation update'
-
-          # Add / update and commit
-          git add ./**/README.md
-          git commit -m 'Automated documentation update [skip ci]' || export NO_UPDATES=true
-
-          # Push
-          if [ "$NO_UPDATES" != "true" ] ; then
-              git push origin "$branch"
-              gh pr create --title "$message" --body "$message"
-          fi
+  publish-docs:
+    environment:
+      name: github-pages
+      url: ${{ steps.deploy-pages.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - run: cd docs && npm ci
+      - uses: actions/configure-pages@v3
+      - run: cd docs && npm pack
+      - uses: actions/upload-pages-artifact@v1
+        with:
+          path: docs/dist
+      - id: deploy-pages
+        uses: actions/deploy-pages@v2
```

Comment:
It would be better to test this workflow in PR to see if it works properly.
Could you please add a pull request trigger and change it so that we can check the artifact?
Like this.
https://github.com/rocker-org/website/blob/e59c051b3b2cfbf2171d3817f3ba8ab86514ccc8/.github/workflows/main.yml#L24-L49